lib/transform-qunit-assertions.js

```diff
@@ -19,6 +19,15 @@ function transformUnaryAssertion(node, prefix) {
   }
 }
 
+function transformBinaryAssertion(node, prefix) {
+  if (node.arguments.length === 2) {
+    var arg1 = node.arguments[0];
+    var arg2 = node.arguments[1];
+    var str = recast.print(arg1).code + ' ' + recast.print(arg2).code;
```

Comment:
The request was to show `equal: foo === bar` for the case `assert.equal(foo, bar);`, but since the [equal assert](https://api.qunitjs.com/equal/) is not using the strict comparison operator, I would show `equal: foo == bar`